appserver/deployment/client/src/main/java/org/glassfish/deployapi/SunDeploymentManager.java

```diff
@@ -363,15 +362,9 @@ private void setWebApplicationTargetModuleIDInfo(TargetModuleID tmid, HostAndPor
     */
     @Override
     public DeploymentConfiguration createConfiguration(DeployableObject dObj) throws InvalidModuleException {
-        try {
-            SunDeploymentConfiguration deploymentConfiguration = new SunDeploymentConfiguration(dObj);
-            deploymentConfiguration.setDeploymentManager(this);
-            return deploymentConfiguration;
-        } catch(ConfigurationException e) {
```

Comment:
The constructor would no longer throw that exception (as it didn't) so it doesn't need handling.

Reply:
I am confirming that.